contracts/src/bridge/Bridge.sol

```diff
@@ -55,14 +62,64 @@ contract Bridge is OwnableUpgradeable, DelegateCallAware, IBridge {
         return outbox;
     }
 
-    function allowedInboxes(address inbox) external view override returns (bool) {
-        return allowedInboxesMap[inbox].allowed;
+    function allowedDelayedInboxes(address inbox) external view override returns (bool) {
+        return allowedDelayedInboxesMap[inbox].allowed;
     }
 
     function allowedOutboxes(address outbox) external view override returns (bool) {
         return allowedOutboxesMap[outbox].allowed;
     }
 
+    modifier onlySequencerInbox() {
+        if (msg.sender != sequencerInbox) revert NotSequencerInbox(msg.sender);
+        _;
+    }
+
+    function enqueueSequencerMessage(bytes32 dataHash, uint256 afterDelayedMessagesRead)
+        external
+        override
+        onlySequencerInbox
+        returns (
+            uint256 seqMessageCount,
```

Comment:
Maybe we should call this return value seqMessageIndex instead of count, as it indicates the index at which the new batch was inserted.